components/tools/OmeroPy/src/omero/plugins/web.py

```diff
@@ -439,16 +460,28 @@ def start(self, args):
             rv = self.ctx.popen(args=django, cwd=location)  # popen
         elif deploy == settings.WSGITCP:
             try:
+                import gunicorn  # NOQA
+            except ImportError:
+                self.ctx.die(608, "Gunicorn not installed.")
+            try:
                 os.environ['SCRIPT_NAME'] = settings.FORCE_SCRIPT_NAME
             except:
                 pass
             cmd = "gunicorn -D -p %(base)s/var/django.pid"
-            cmd += " -b %(host)s:%(port)s -w 5"
+            cmd += " --bind %(host)s:%(port)s"
+            cmd += " --workers %(workers)s "
+            cmd += " --worker-connections %(worker_conn)s"
```

Comment:
Could `port`/`workers`/`worker_conn` could be tightened up from `%()s` to `%()d`?